client/app/house/components/MembersManagementModal.tsx

```diff
@@ -33,10 +35,17 @@ const MembersManagementModal = ({
   const { toast } = useToast()
   const { onUpdate, isPending: isUpdating } = useUpdateAdmin()
   const houseId = house.houseId
-  const [firstMember, ...restMembers] = house.houseMembers
-  const sortedRestMembers = restMembers.sort((a, b) => a.id - b.id) // Sorted by userId to fix the order
-  const houseMembers = [firstMember, ...sortedRestMembers]
-  const userId = houseMembers[0].id
+
+  const loggedInMember = house.houseMembers.find(
+    (member) => member.id === userId,
+  )!
```

Comment:
I used not null assertion here becuase there is no possibility that loggedInMember becomes undefined or null